cogs/tickets.py

```diff
@@ -190,6 +190,12 @@ async def close(self, ctx, *, reason: str = None):
         await log_channel.send(embed=embed,
                                file=discord.File(f"transcripts/{ctx.channel.name}.html", "transcript.html"))
 
+    @commands.command(hidden=True, aliases=['sc', 'staffclaim'])
```

Comment:
Could possibly use some other aliases that are easier to guess, along the lines of "renameticket" or something.